cylc/flow/suite_files.py

```diff
@@ -636,7 +711,7 @@ def _dump_item(path, item, value):
 def _get_cache_dir(reg, owner, host):
     """Return the cache directory for remote suite service files."""
     return os.path.join(
-        UserFiles.get_path(),
+        os.path.expanduser("~"), ".cylc", "auth"
         "%s@%s" % (owner, host), reg
     )
```

Comment:
All of the `_get_cache_dir` uses can now be safely stripped out (as we are no longer creating the cache directory anyway), we can clean this up in #3444.

Reply:
Yes, I'll make mental note of that for the tidy up PR.